src/commands/typescript/update.ts

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as path from 'path';
+import { flags, FlagsConfig, SfdxCommand } from '@salesforce/command';
+import { fs, Messages, SfdxError } from '@salesforce/core';
+import { exec } from 'shelljs';
+import { set } from '@salesforce/kit';
+import { asObject, ensureString, get } from '@salesforce/ts-types';
+import { NpmPackage, Package } from '../../package';
+import { SinglePackageRepo } from '../../repository';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-release-management', 'typescript.update');
+
+export default class Update extends SfdxCommand {
+  public static readonly description = messages.getMessage('description');
+  public static readonly flagsConfig: FlagsConfig = {
+    version: flags.string({
+      char: 'v',
+      description: messages.getMessage('typescriptVersion'),
+    }),
+    target: flags.string({
+      char: 't',
+      description: messages.getMessage('esTarget'),
+      default: 'ESNext',
+    }),
+  };
+
+  public async run(): Promise<void> {
```

Comment:
Would it be helpful to return a result?  Will this be used from CI and the result could be helpful in the reporting?